### DISCUSSION TITLE
Vue 3x Compatibility

### DISCUSSION THREAD

**Discussion by Silbad** [bootstrap-vue/bootstrap-vue#6495] [General]
Hello,
When do you expect to be compatible with Vue 3x & an update for the [vue-cli plugin](https://github.com/GregYankovoy/vue-cli-plugin-bootstrap-vue) ?
Thanks

**Comment by Hiws** [comment]
There's a pinned issue regarding Vue 3 support. https://github.com/bootstrap-vue/bootstrap-vue/issues/5196
And the vue-cli plugin is managed by someone else, so probably better to ask over there.
